.gitignore

```diff
@@ -5,3 +5,5 @@ cmake-*
 build
 .DS_Store
 *.pdf
+spi/w5500_spi/mpu9250_spi.fzz
+spi/w5500_spi/mpu9250_spi_bb.png
```

Comment:
No need to add these to `.gitignore`, just delete them from your local directory :smiley:

Reply:
I deleted the .gitignore on my local.

Reply:
You've also (accidentally?) deleted it from this repo too, which you don't want to do.